bin/alluxio

```diff
@@ -398,9 +398,13 @@ function main {
     done
   ;;
   "grpcGen")
+    rm -rf ${BIN}/../core/transport-options/src/main/java/alluxio/grpc
+    for src_file in $(find ${BIN}/../core/transport-options/src/grpc -name '*.proto' -type f); do
+      protoc --java_out=${BIN}/../core/transport-options/src/main/java --proto_path=${BIN}/../core/transport-options/src/grpc --plugin=protoc-gen-grpc-java=${BIN}/protoc-gen-grpc-java-1.10.1-osx-x86_64.exe --grpc-java_out=${BIN}/../core/transport-options/src/main/java ${src_file}
+    done
     rm -rf ${BIN}/../core/transport/src/main/java/alluxio/grpc
     for src_file in $(find ${BIN}/../core/transport/src/grpc -name '*.proto' -type f); do
-      protoc --java_out=${BIN}/../core/transport/src/main/java --proto_path=${BIN}/../core/transport/src/grpc --plugin=protoc-gen-grpc-java=${BIN}/protoc-gen-grpc-java-1.10.1-osx-x86_64.exe --grpc-java_out=${BIN}/../core/transport/src/main/java ${src_file}
+      protoc --java_out=${BIN}/../core/transport/src/main/java --proto_path=${BIN}/../core/transport/src/grpc --proto_path=${BIN}/../core/transport-options/src/grpc --plugin=protoc-gen-grpc-java=${BIN}/protoc-gen-grpc-java-1.10.1-osx-x86_64.exe --grpc-java_out=${BIN}/../core/transport/src/main/java ${src_file}
```

Comment:
is there a better way than referencing `${BIN}/protoc-gen-grpc-java-1.10.1-osx-x86_64.exe`?

Reply:
Making maven to generate the files maybe?

Reply:
i would prefer pre-generated files. we can defer this to following PRs

Reply:
I didn't mean generating during the build but invoking the generation phase only when we want to regenerate, not triggering the generation in the regular build.

Reply:
I put this on my task list for the consolidation work.